src/server/routes/generators/typescript.ts

```diff
@@ -0,0 +1,22 @@
+import { FastifyInstance } from 'fastify'
+import ejs from 'ejs'
+import { PostgresMeta } from '../../../lib'
+import template from '../../../lib/templates/typescript'
+import { DEFAULT_POOL_CONFIG } from '../../constants'
+
+export default async (fastify: FastifyInstance) => {
+  fastify.get<{
+    Headers: { pg: string }
+  }>('/', async (request, reply) => {
+    const connectionString = request.headers.pg
+    const responseFormat = request.headers.responseFormat || 'text/plain'
+
+    const pgMeta = new PostgresMeta({ ...DEFAULT_POOL_CONFIG, connectionString })
+
+    const { data: tables } = await pgMeta.tables.list({})
+    const { data: functions } = await pgMeta.functions.list({})
+    const definition = ejs.render(template, { tables, functions })
```

Comment:
pgMeta already has the data in JSON format, so we have done the hard work - we just need to pass the data to the template